cmd/hard/main.go

```diff
@@ -0,0 +1,280 @@
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var downCmd = &cobra.Command{
+	Use:   "down",
+	Short: "Stop the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "down")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var restartCmd = &cobra.Command{
+	Use:   "restart",
+	Short: "Restart the environment",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "restart")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var psCmd = &cobra.Command{
+	Use:   "ps",
+	Short: "Display the status of all containers",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "ps")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Run()
+	},
+}
+
+var bashCmd = &cobra.Command{
+	Use:   "bash",
+	Short: "Start a shell session within the app container",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "exec", "app", "bash")
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Stdin = os.Stdin
+		dcCmd.Run()
+	},
+}
+
+var laravelCmd = &cobra.Command{
+	Use:   "laravel",
+	Short: "Run Laravel commands",
+	Run: func(cmd *cobra.Command, args []string) {
+		dcCmd := dockerComposeCmd()
+		dcCmd.Args = append(dcCmd.Args, "exec", "app", "laravel")
+		dcCmd.Args = append(dcCmd.Args, args...)
+		dcCmd.Stdout = os.Stdout
+		dcCmd.Stderr = os.Stderr
+		dcCmd.Stdin = os.Stdin
+		dcCmd.Run()
+	},
+}
+
+func runInProject(projectCmd string, args []string) {
+	project := ""
+	if len(args) > 0 {
+		if _, err := os.Stat(filepath.Join(wwwPath, args[0])); err == nil {
+			project = args[0]
+			args = args[1:]
+		}
+	}
+
+	if project == "" {
+		dir, _ := os.Getwd()
+		project = filepath.Base(dir)
+	}
+
+	dcCmd := dockerComposeCmd()
+	dcCmd.Args = append(dcCmd.Args, "exec", "-it", "--workdir", "/var/www/"+project, "app", projectCmd)
+	dcCmd.Args = append(dcCmd.Args, args...)
+	dcCmd.Stdout = os.Stdout
+	dcCmd.Stderr = os.Stderr
+	dcCmd.Stdin = os.Stdin
+	dcCmd.Run()
+}
+
+var phpCmd = &cobra.Command{
+	Use:   "php",
+	Short: "Run PHP commands",
+	Run: func(cmd *cobra.Command, args []string) {
+		runInProject("php", args)
+	},
+}
+
+var composerCmd = &cobra.Command{
+	Use:   "composer",
+	Short: "Run Composer commands",
+	Run: func(cmd *cobra.Command, args []string) {
+		runInProject("composer", args)
+	},
+}
+
+var nodeCmd = &cobra.Command{
+	Use:   "node",
+	Short: "Run Node commands",
+	Run: func(cmd *cobra.Command, args []string) {
+		runInProject("node", args)
+	},
+}
+
+var npmCmd = &cobra.Command{
+	Use:   "npm",
+	Short: "Run npm commands",
+	Run: func(cmd *cobra.Command, args []string) {
+		runInProject("npm", args)
+	},
+}
+
+var yarnCmd = &cobra.Command{
```

Comment:
Not working properly. Hard cli don't what to do with "-" flag

![image](https://github.com/user-attachments/assets/ca3c6de0-26b8-4d96-bda9-e4f6f301519e)

Reply:
Now all command flags are passed to the command within docker